db/seeds.rb

```diff
@@ -0,0 +1,46 @@
+melanie = User.create(username: 'melanie', password: 'password', email: 'melaniegzmail.com')
+sherianne = User.create(username: 'sherianne', password: 'password', email: 'sherianne@gmail.com')
+allain = User.create(username: 'allain', password: 'password', email: 'allain@j.com')
+jenna = User.create(username: 'jenna', password: 'password', email: 'jenna@gmail.com')
+jung = User.create(username: 'jung', password: 'password', email: 'jung@gmail.com')
+natalie = User.create(username: 'natalie', password: 'password', email: 'natalie@gmail.com')
+
+dog = Question.create(title: 'Dog question', body: "Do you like dogs?", user: melanie)
```

Comment:
here, 'user' is actually meant to be a user_id integer. when we use t.references, it is a way for activerecord to automatically create a foreign key for us in the table that is referenced. so if questions references user, it is actually referencing a user ID.